### PR TITLE
Power Animated Button

### LINKED ISSUE #1131: [Add] POWER BUTTON WITH ANIMATED ICON
### Details

I would like to add a Power Button with an animated icon. Watch the below video :-

### Screenshots

https://github.com/Spyware007/Animating-Buttons/assets/100849696/31347f43-c7b8-457f-8c8b-11a0f1f874d5

### Type of Contribution

- [ ] Update to an existing Animated Button
- [X] Adding a new Button
- [ ] Resolving a bug
- [ ] Proposal to the Repository
- [ ] Changes related to documentation or README.md
- [ ] Other Changes

### Checklist

- [X] I have checked the existing [issues](https://github.com/Spyware007/Animating-Buttons/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/Spyware007/Animating-Buttons/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by sabhisharma-ise**
## Fixes Issue

Closes #1131 

## Changes proposed
Power button is added to the website

## Screenshots
image

## Checklist

- [X] You've linked this PR to the correct issue.
- [X] You have checked that the code is working correctly.
- [X] You ⭐️ the repository!

### CODE REVIEW

**Comment by Spyware007** [comment]
# Please check #1346

File: src/Data.js
Comment (adity2911) [inline src/Data.js:1]: Why did you delete data of so many buttons?

**Comment by adity2911** [comment]
The PR cant be accepted as the data of other buttons seems to be deleted. Fix it and then only we can merge the PR.

**Comment by sabhisharma-ise** [comment]
@adity2911 hey, can you check now please!

**Review by adity2911 — APPROVED** [review]
